CITATION.cff

```diff
@@ -0,0 +1,20 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: unknown data framework
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
```

Comment:
The following authors are missing:  
* Lu Gan (https://www.gesis.org/en/institute/about-us/staff/person/Lu.Gan)
* Sebastian Tiesler (https://www.ibi.hu-berlin.de/de/institut/personen/tiesler/)
* Yousef Younes (https://orcid.org/0000-0003-1271-3633)

Reply:
Thank you for the quick reply and for the information about the missing authors :)